.github/workflows/Collect.yaml

```diff
@@ -53,7 +55,7 @@ jobs:
               }
               $settingsFile = Join-Path $settings.baseFolder "deploy.json"
               $settings | ConvertTo-Json | Set-Content $settingsFile -Encoding UTF8
-              . ".\Internal\Collect.ps1" -configName $settingsFile -githubOwner $githubOwner -token $token -directCOMMIT:('${{ github.event.inputs.directCOMMIT }}' -eq 'Y')
+              . ".\Internal\Collect.ps1" -configName $settingsFile -githubOwner $githubOwner -token $token -directCommit:($ENV:_directCommit -eq 'true'- or $ENV:_directCommit -eq 'Y')
```

Comment:
If we do not keep it for a while, then every single development branch you have will stop working after you update the branch and until you update AL-Go System Files (which you cannot do because the yaml file will send a 'Y' to the action).
We can remove these checks in the future.

Reply:
Let's say I have some repository on freddydk/AL-Go@ALDoc - and this branch is Work in progress.
Now we merge this PR and update the ALDoc branch with the changes.
If we don't support Y and N, then all my test repos using any of my branches will be dead in the water.
Same if any other partner has a branch which they use.